nomos-bundler/README.md

```diff
@@ -0,0 +1,16 @@
+# Nomos Bundler
+
+This crate performs the bundling tasks for other Nomos' crates.
+
+## Usage
+
+Any crate that needs to be bundled should have their own file in the `src` directory of this crate,
+with the same name as the bundled crate, and implement their bundling logic.
```

Comment:
> with the same name as the bundled crate

Considering our current directory structure: `nomos-bundler/nomos-cli/bundle.rs`,

I think it might be clearer to say like below. What do you think about this?

```suggestion
Any crate that needs to be bundled should have their own directory in the `src` directory of this crate,
with the same name as the bundled crate, and it should contain a file that implements their bundling logic.
```

Reply:
It's indeed much better phrased the way you put it.

That being said, you made me rethink if it's a good idea to be this specific: I feel like when working on the remaining bundles we will arrive at a "single bundling binary" solution. I don't really know, though.

Reply:
It would be good if we can have the single bundling binary somehow but I'm totally fine with the current approach implemented in this PR :)